### ISSUE TITLE
enable code linting for python files using black-flake8

### ISSUE THREAD

**Issue by wrathdev** [wrathdev/vpnhelper#13]
Adding code linting will be helpful in identifying some common and uncommon mistakes that are made during coding.
Also It will lead to consistent code workflow for all devs.

I choose flake8and black just because I am more familiar with them. Suggest if you know better alternative

**Comment by MarioHdpz** [comment]
I can do this one. But I suggest to also add a something to manage packages better.

Could be a package manage like poetry or pipenv, or may be a requirements.txt

**Comment by wrathdev** [comment]
Yeah that would be great. I think since it's just a utility script so requirements.txt would be enough.
Thanks
